notebook/agentchat_qdrant_RetrieveChat.ipynb

```diff
@@ -0,0 +1,3315 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/microsoft/autogen/blob/main/notebook/agentchat_qdrant_RetrieveChat.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"toc\"></a>\n",
+    "# Using RetrieveChat with Qdrant for Retrieve Augmented Code Generation and Question Answering\n",
```

Comment:
Have you ran all the code in the notebook?

Reply:
I think it's OK to have only one or two examples for using qdrant database.

Reply:
Looks like the outputs were not commited. Will do.

Reply:
https://github.com/microsoft/autogen/pull/303/commits/6f6b205b2ab7da1571cfab1805c31d887f649c9a

Reply:
I think the questions are not appropriate for the notebook since the old notebook was based on FLAML when AutoGen was a subpackage of FLAML.

Reply:
Or updating the context file generation instruction in the comment.

Reply:
FYI, I made a PR for updating RetrieveChat notebook https://github.com/microsoft/autogen/pull/379

Reply:
> FYI, I made a PR for updating RetrieveChat notebook #379

Updated accordingly. https://github.com/microsoft/autogen/pull/303/commits/b15fe5ab6e39e1278adb174a5a1f3789c6839a92.